src/NUnitFramework/framework/Internal/Execution/WorkItem.cs

```diff
@@ -177,6 +177,11 @@ public virtual ParallelExecutionStrategy ExecutionStrategy
                 return _executionStrategy.Value;
             }
         }
+
+        /// <summary>
+        /// Indicates whether this work item should use a separate dispatcher.
+        /// </summary>
+        public virtual bool IsolateChildTests { get; } = false;
```

Comment:
We can save an autogenerated field per instance by doing `IsolateChildTests => false;` since we aren't setting `IsolateChildTests` in a constructor.

Reply:
I don't love this syntax, especially when the property is virtual. IMO, the existing syntax is clearer to read and avoids confusion.

Reply:
This is theoretical but C# 7.0 allows `IsolateChildTests { get => false; }` as well, if you still want to use the `get` keyword without allocating a field.